opg-php-fpm-1604/Dockerfile

```diff
@@ -26,6 +26,13 @@ ADD  docker/logrotate.d/app /etc/logrotate.d/app
 ADD  docker/my_init.d /etc/my_init.d
 RUN  chmod a+x /etc/my_init.d/*
 
+ADD  docker/php/* /etc/php/7.0/
+
+# disable php-xdebug by default
+RUN  rm -f /etc/php/7.0/fpm/conf.d/*xdebug*
```

Comment:
@iterative-it the new images need to be added to the Makefile in the root of this repo